### PR TITLE
TPS renamed to CIPS

### PR DESCRIPTION
**Pull request by gsr-shanks**
TPS.next project has been renamed to CIPS.
This PR addresses the change in name.

### CODE REVIEW

File: resultsdbupdater/utils.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-import uuid
+import logging
 import re
+import uuid
 
+import fedmsg
 import requests
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
-import fedmsg
```
Comment (mprahl) [inline resultsdbupdater/utils.py]: Why did this line change?

File: tests/fake_messages/cips_message.json [hunk 2]
```diff
@@ -20,41 +20,41 @@
       "results": {
         "start_time": "2017-11-29T08:53:40+00:00",
         "end_time": "2017-11-29T08:58:19+00:00",
-        "tps_report": "https://server.domain.local/job/test-package-sanity-development/label=ose-slave-tps,provision_arch=x86_64/734//artifact/tps-next/reports/results.html",
-        "tps_status": "PASSED"
+        "cips_report": "https://server.domain.local/job/test-package-sanity-development/label=ose-slave-tps,provision_arch=x86_64/734//artifact/tps-next/reports/results.html",
```
Comment (mprahl) [inline tests/fake_messages/cips_message.json]: The URL here should probably change to what CIPS would actually provide.

File: resultsdbupdater/consumer.py [hunk 3]
```diff
@@ -1,9 +1,8 @@
-import fedmsg.consumers
-import fedmsg.config
+import fedmsg
```
Comment (mprahl) [inline resultsdbupdater/consumer.py]: This change in the import causes the the class definition on line 12 to fail (you can see this in the CI run). Please revert the imports back.

Comment (gsr-shanks) [reply resultsdbupdater/consumer.py]: Ack

**Pull request closed without merging**